### DISCUSSION TITLE
Is it possible to setup locale for gcc in msys2?

### DISCUSSION THREAD

**Discussion by SherkeyXD** [msys2/MSYS2-packages#4552] [Q&A]
I didn't see any `gcc-<version>-locales` packages (`gettext` is avaible though).

**Comment by Biswa96** [comment]
Could you explain the actual issue? Does any Linux distribution provide that kind of package?

**Reply by SherkeyXD** [reply]
> Could you explain the actual issue? 

I've done settings with my locale, and it looks all good. Then I noticed that gcc outputs are still in English.

> Does any Linux distribution provide that kind of package?

As I know, debian provides it.

![Screenshot_2024-04-26-00-55-42-324_org.connectbot-edit.jpg](https://github.com/msys2/MSYS2-packages/assets/57581480/935f6edd-2f62-4eee-9c04-2ebb4ba4f172)

**Comment by Biswa96** [comment]
msys gcc package provide the locale files but mingw gcc does not. Let me discuss this with others.
